config/config.go

```diff
@@ -110,6 +110,7 @@ type Config struct {
 	Migration             Migration         `json:"migration"`
 	SMController          SMController      `json:"smController"`
 	UMController          UMController      `json:"umController"`
+	DisableLogsTrim       bool              `json:"disableLogsTrim"`
```

Comment:
Should name `TrimAMQPDataLogs" and should be false by default.

Reply:
Applied suggestion, but made default value true